resources/language/resource.language.es_es/strings.po

```diff
@@ -126,11 +126,11 @@ msgstr "Biblioteca"
 
 msgctxt "#30026"
 msgid "Enable custom library folder"
-msgstr "Habilitar carpeta personalizada de la biblioteca"
+msgstr "Habilitar carpeta para biblioteca personalizada"
 
 msgctxt "#30027"
 msgid "Custom library path"
-msgstr "Ruta personalizada de la biblioteca"
+msgstr "Ruta biblioteca personalizada"
```

Comment:
If this text fits in the dialog box, please use 'Ruta **a la** biblioteca personalizada' instead. If not, that message string is fine.

Reply:
We leave it as 'Ruta' ??

Reply:
And if we leave it with just the word "Path/Ruta".